packages/core/strapi/lib/services/entity-service/index.js

```diff
@@ -164,7 +188,7 @@ const createDefaultImplementation = ({ strapi, db, eventHub, entityValidator })
 
     await this.emitEvent(uid, ENTRY_CREATE, entity);
```

Comment:
should this not also use this.wrapResult() ? since now wat the api returns and what event hub returns are 2 fully different things aka audit logs will not see the same entity as the api

Reply:
indeed :) wrapResult should also wrap the entity emited in this event, good catch!